build.gradle

```diff
@@ -47,6 +47,10 @@ allprojects {
         if (project.hasFlags('coverage')) {
             systemProperty 'com.linecorp.armeria.testing.coverage', 'true'
         }
+        // Pass special environment variable to enable demo mode for `*DocServiceTest`.
+        if (rootProject.hasProperty('demo')) {
```

Comment:
nit: Mostly LGTM, but I am worried `demo` has a too wide meaning. How about renaming to  `docservice.demo`?